### PR TITLE
#219 - packages attribute in primary

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #219: `XmlPrimary` is not responsible for counting packages and setting correct value to `packages` attribute, `Metadata` is, it's done in `brush` method. I've added corresponding assertion to the test and removed todo.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#221](https://github.com/artipie/rpm-adapter/issues/221) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/8ef3fcf5-5b74-4348-8094-862e6bea63f6); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/rpm/pkg/ModifiableMetadataTest.java [hunk 1]
```diff
@@ -76,12 +76,13 @@ void generatesMetadataFile(@TempDir final Path temp) throws IOException {
...
         MatcherAssert.assertThat(
-            "Has 'abc' and 'nginx' packages",
+            "Has 'abc' and 'nginx' packages, writes `packages` attribute correctly",
             new String(Files.readAllBytes(res), StandardCharsets.UTF_8),
             XhtmlMatchers.hasXPath(
                 //@checkstyle LineLengthCheck (2 lines)
                 "/*[local-name()='metadata']/*[local-name()='package']/*[local-name()='name' and text()='abc']",
-                "/*[local-name()='metadata']/*[local-name()='package']/*[local-name()='name' and text()='nginx']"
+                "/*[local-name()='metadata']/*[local-name()='package']/*[local-name()='name' and text()='nginx']",
+                "/*[local-name()='metadata' and @packages='2']"
```
Comment (paulodamaso) [inline src/test/java/com/artipie/rpm/pkg/ModifiableMetadataTest.java:85]: @olenagerasimova Just to make reading easier, could you please put this `xpath` query on top of the others? It seems more logical that we assert first if some element attributes are correct, then if the children elments are.

**Review by paulodamaso — CHANGES_REQUESTED** [review]
@olenagerasimova Just one comment, please take a look

**Comment by olenagerasimova** [comment]
@paulodamaso thanks, corrected, take a look please

**Comment by olenagerasimova** [comment]
@paulodamaso ping

**Review by paulodamaso — APPROVED** [review]
@olenagerasimova Great, thank you!

**Comment by paulodamaso** [comment]
@g4s8 please merge

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by olenagerasimova** [comment]
@g4s8 ping

**Comment by olenagerasimova** [comment]
@g4s8 could you please approve or merge this?

**Review by g4s8 — APPROVED** [review]

**Comment by sereshqua** [comment]
@paulodamaso ping

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by paulodamaso** [comment]
@sereshqua Sorry for the delay, i'll try to find at least 3 issues in the next CRs

**Comment by paulodamaso** [comment]
@g4s8 please take a look at this, @0crat lost rtack of this one after QA

**Comment by 0crat** [comment]
Are you speaking to me or about me [here](https://github.com/artipie/rpm-adapter/issues/221#issuecomment-644906035); you must always start your message with my name if you want to address it to me, see [§1](http://www.zerocracy.com/policy.html#1)

**Pull request merged**